### DISCUSSION TITLE
Any reason intrinsic heavy methods aren't marked with AggressiveInlining

### DISCUSSION THREAD

**Discussion by john-h-k** [SixLabors/ImageSharp#1241] [Q&A] [answered]
Looking through some AVX methods and I notice they aren't marked with it. Often intrinsic methods can have deceivingly large IL which results in the JIT failing to inline them when it is actually profitable. Is there a specific reason it isn't applied to hint the JIT in the right direction?

**Comment by JimBobSquarePants** [comment, marked as answer]
No reason, just would have been missed.
